astropy/coordinates/tests/test_solar_system.py

```diff
@@ -33,11 +33,11 @@
 
 @pytest.mark.remote_data
 @pytest.mark.skipif(not HAS_SKYFIELD, reason='requires skyfield')
-def test_positions_skyfield(tmpdir):
+def test_positions_skyfield(tmp_path):
     """
     Test positions against those generated by skyfield.
     """
-    load = Loader(tmpdir)
+    load = Loader(tmp_path)
```

Comment:
I am not familiar with skyfield Loader object, so I'll have to assume that green CI with remote data got this one covered...